script.js

```diff
@@ -0,0 +1,60 @@
+
+     
+    let pic = document.getElementById('cnvs');
+    let ctx= pic.getContext('2d');
+
+    changeCanvas(pic.width, pic.height)
+
+
+    function firstPic(matrix, n, typeOfColor){
```

Comment:
again, the function name could be better. as I see it draws an image given by some data. maybe drawImage?